DIPs/DIP1016.md

```diff
@@ -175,6 +175,32 @@ Again where initial construction of `result` should be performed at the moment o
 
 It is important that `T` be defined as the _parameter_ type, and not `auto` (ie, the argument type), because it will allow implicit conversions to occur naturally, with identical behavior as when the parameter is not `ref`. The user should not experience edge cases, or differences in functionality when calling `fun(int x)` vs `fun(ref int x)`.
 
+### `@property` and implicit wideing casts
+
+`@property` and implicit wideing casts are syntactically identical to public feilds and variables respectively and so a change in the API
+from one to the other would result in silent change to the users code. 
+For this reason, they promote to`const ref` rather than `ref`, since this guaruntees that the caller is not expecting the value to change as a result of the call. 
+All other cases (i.e. not `@property` or implicit wideing casts) the expressions that give rise to the promoted rvalue-to-ref do not outlive the scope of the call expression so there is no risk of such use after expected modification.
```

Comment:
All other cases  -> In all other cases

) the -> ), the

expression so -> expression, so